OperatingSystem-Services/Service-SOC/ServiceDocumentation/SuricataIntegrationWazuh.md

```diff
@@ -0,0 +1,136 @@
+# Integration of Suricata into Wazuh
```

Comment:
Rename title maybe - `Wazuh - Suricata Integration`

Is this docker specific or no